pokedex-with-bit/requirements.txt

```diff
@@ -1,3 +1,3 @@
 click==7.1.2
-tensorflow-cpu==2.1
-jina[hub,http,cv]==1.0.5
\ No newline at end of file
+tensorflow-cpu==2.5
+jina[hub,http,cv]==2.0.0rc2
```

Comment:
I don't believe this works

Reply:
It should not produce any errors when installing the packages. It is important to use python3.8, because in python3.7 I think the tensorflow version is not available.

Reply:
Ah your right, sorry about that